pkg/apis/sippy/v1/types.go

```diff
@@ -50,10 +50,11 @@ type MinimumPassRatesByComponent struct {
 // FailingTestBug describes a single instance of failed test with bug or failed test without bug
 // differs from failingtest in that it includes pass rates for previous days and latest days
 type FailingTestBug struct {
-	Name      string              `json:"name"`
-	Url       string              `json:"url"`
-	PassRates map[string]PassRate `json:"passRates"`
-	Bugs      []bugsv1.Bug        `json:"bugs,omitempty"`
+	Name           string              `json:"name"`
+	Url            string              `json:"url"`
+	PassRates      map[string]PassRate `json:"passRates"`
+	Bugs           []bugsv1.Bug        `json:"bugs,omitempty"`
+	AssociatedBugs []bugsv1.Bug        `json:"associatedBugs,omitempty"`
```

Comment:
The distinction between `bugs` and `associatedBugs` isn't immediately obvious at an API level.  `nonCausalBugs`?

Reply:
i dunno that noncausalbugs is going to be any more obvious.  would you settle for me adding a comment that explains the distinction?